### ISSUE TITLE
Brunch does not ignore emacs backup files

### ISSUE THREAD

**Issue by nikrou** [brunch/brunch#911]
Hi,

I'm sorry if it's not the right place. I cannot figure where to post and how to solve it.

I'm using brunch version 1.7.20 with a config file like that:
```javascript
exports.config = {
	paths: {
		public: "."
	},
	files: {
		javascripts: {
			joinTo: {
				'js/app.js': /^app/,
				'js/vendor.js': /^(?!app)/
			}
		},
		stylesheets: {
			joinTo: 'css/app.css'
		},
		templates: {
			joinTo: 'js/app.js'
		}
	}
};
```
In the app tree, I've got a backup file generated by emacs : router.js~
My file is to ignore and brunch failed with error:
```shell
 26 Jan 14:38:27 - error: Reading of 'router.js' failed. ENOENT, open 'router.js'
```

I try to use config.conventions.ignored but it's not seems to do the job.
Thanks in advance for any help.

**Comment by es128** [comment]
This has to do with the new `atomic` setting in chokidar, which you should be able to disable in your config with `watcher: atomic: false` in your config, but I'm concerned that may lead to other unwanted side effects.

I need to consider how to deal with the case of these stale backup files in a better way by default.

**Comment by nikrou** [comment]
I'm not sure to really understand. I try to build my application using --debug but it doesn't give me more helping informations.
Use "watcher: atomic: false" does not solve my issue. I use for now, "watcher: usePolling: true". It works but I dislikes to do not understand things.
If you need more information, just tell me.

**Comment by es128** [comment]
The `atomic` setting automatically makes files like `router.js~` emit as `router.js` because of the way some text editors behave. Usually these sorts of backup files exist only momentarily during the save process, but in a case like yours where they stay around for longer, this creates a problem. It's something that needs to be fixed within chokidar, which is Brunch's file-watching module.

The `atomic` setting is only on by default when watching with the non-polling `fs.watch`-backed mode, so setting `usePolling` as you have done would fix it, at the expense of heavier CPU utilization due to polling-mode watching.

I also should check whether Brunch properly passes all options defined in `watcher` to chokidar.

**Comment by nikrou** [comment]
Thanks es128 for explanations.

**Comment by es128** [comment]
Resolved in brunch master branch and 1.8.0-pre via chokidar 1.0.0-rc3
